### ISSUE TITLE
 Android Phone cannot read the device's name

### ISSUE THREAD

**Issue by hwra2008** [pauldemarco/flutter_blue#519]
Most of them are ids ,but the native way with Android is to get the name.
I could not ios.So how did flutter get its name when I wanted to ask you for help
Many thanks.

**Comment by pauldemarco** [comment]
Hi @hwra2008,
I'm not sure I understand the question, can you rephrase?

You can also ping me directly through the discord here:
http://discord.gg/Yk5Efra

**Comment by qinhongfei** [comment]
I also ran into this problem. I couldn't get the Bluetooth device name when running on an Android phone
